### PR TITLE
add the UFRGS admissions dataset

### PR DESCRIPTION
**Pull request by olliethomas**

### CODE REVIEW

File: ethicml/data/tabular_data/admissions.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+from ..dataset import Dataset
+from ..util import deprecated
+
+__all__ = ["Admissions", "admissions"]
+
+AdmissionsSplits = Literal["Gender"]
+
+
+@deprecated
```
Comment (thomkeh) [inline ethicml/data/tabular_data/admissions.py:51]: @olliethomas why do you keep adding deprecated code? 😄 deprecated code is there to be removed! not added!

Comment (thomkeh) [reply ethicml/data/tabular_data/admissions.py:51]: or are you trying to send the hidden message that you prefer the capitalized version of the function?

Comment (olliethomas) [reply ethicml/data/tabular_data/admissions.py:51]: only because I don't know what's happening with the uppercase vs lowercase.

Comment (olliethomas) [reply ethicml/data/tabular_data/admissions.py:51]: > or are you trying to send the hidden message that you prefer the capitalized version of the function?
I think it's safe to assume that there is no hidden message and that I'm mostly just copying the adult dataset file ever time I add a new dataset

**Pull request merged**